src/App.test.tsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
+
 import { render, screen } from '@testing-library/react';
-import App from './App';
+
+import App from './App/App';
```

Comment:
в папке ./App делаешь файл index и импортишь не 
```suggestion
import App from './App';
```